### PR TITLE
CFD-296 Home page : Suggested Groups (Logged-in users) 

### PR DESCRIPTION
**Pull request by HelenaEksler**
#479

### CODE REVIEW

**Comment by HelenaEksler** [comment]
isaacnewton user:
selection_213

admin user:
selection_214

@nader77 PR is ready for review. Design will be fixed in theming issue.

**Comment by nader77** [comment]
@HelenaEksler Looks good, but my groups plugin is under ``c4m_features_homepage``, Don't you think this 2 plugins should be in the same directory ?

File: capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module [hunk 1]
```diff
@@ -68,3 +68,61 @@ function c4m_content_group_preprocess_block(&$variables) {
...
+
+/**
+ * Helper function to render group details.
+ * @return string
+ */
+function c4m_content_group_c4m_suggested_groups_get_groups() {
+  global $user;
+
+  $account = user_load($user->uid);
+  $user_wrapper = entity_metadata_wrapper('user', $account);
```
Comment (amitaibu) [inline capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]: $user_wrapper = entity_metadata_wrapper('user', $user->uid);

**Comment by HelenaEksler** [comment]
@zero2one Travis if failing because of the error in one test 
selection_225
And I can't find the reason for this. Maybe you've got this error in some other place?

**Comment by zero2one** [comment]
@nader77 had the same issue in a test last week. He fixed that one on sunday.

Can you ask him?

**Comment by HelenaEksler** [comment]
For him it has just disappeared after rerunning Travis. For me it doesn't work for some reason.

**Comment by nader77** [comment]
@zero2one Tests are green, PR is ready for review.

**Pull request merged**